.travis.yml

```diff
@@ -47,9 +47,15 @@ stages:
 jobs:
   include:
     - name: "Coverage"
+      if: branch != master
```

Comment:
Let's do this always and remove JaCoCo from the job below.

Reply:
OK